core/src/main/scripts/importer/validateData.py

```diff
@@ -1370,15 +1401,16 @@ def checkHeader(self, cols):
                     # store original property as found in file
                     if value != srv_attr_properties[attr_property]:
                         self.attr_defs_overridden[col_index][attr_property] = value
-                        self.logger.warning(
-                            "%s definition for attribute '%s' does not match "
-                            "the portal, and will be loaded as '%s'",
-                            attr_property,
-                            col_name,
-                            srv_attr_properties[attr_property],
-                            extra={'line_number': self.attr_defs[col_index].keys().index(attr_property) + 1,
-                                   'column_number': col_index + 1,
-                                   'cause': value})
+                        if not self.relaxed_mode and not self.fill_in_attr_defs:                            
+                            self.logger.warning(
+                                "%s definition for attribute '%s' does not match "
+                                "the portal, and will be loaded as '%s'",
+                                attr_property,
+                                col_name,
+                                srv_attr_properties[attr_property],
+                                extra={'line_number': self.attr_defs[col_index].keys().index(attr_property) + 1,
+                                       'column_number': col_index + 1,
+                                       'cause': value})
```

Comment:
This will skip over an important validation if a data file does contain the attribute definitions while the flag is given.